### ISSUE TITLE
Backward incompatible change by 659ed15c59e460578d571b2ae246d72d29835035

### ISSUE THREAD

**Issue by leoliu** [pqina/filepond#501]
Hello fellow developer!

### Summary

I have been running always up-to-date filepond for 2 years and haven't seen any big issues for at least a year until `4.13.6`.

### How to reproduce

I have a callback which remove uploaded file from the dropping area if upload succeeds like this:

```js
pond.on("processfile", (error, file) => {
      if (!error) pond.removeFile(file);
    })
```

This is now completely broken in `4.13.6`.

### Expected behaviour

...

### Additional information

...

| Environment  | Version
|------------- | -----------
| OS           | MacOS / Windows / iOS / Linux / ...
| Device       | iPhone SE / Galaxy Note 9 / ...
| Browser      | Safari 11.1.1 / Edge 15 / ...

**Comment by rikschennink** [comment]
> This is now completely broken

That’s not a bug description, provide more details, the issue template is there for a reason.

Closed for now.

**Comment by leoliu** [comment]
Sorry for the unhelpful bug report. Will come back to do it after finishing the work at hand.

**Comment by rikschennink** [comment]
Probably fixed by `4.13.7`

**Comment by leoliu** [comment]
Indeed it's fixed in `4.13.7`. Thanks.
